doc/jsdoc.json

```diff
@@ -7,6 +7,7 @@
       "lib/metadata",
       "lib/client.js",
       "lib/execution-profile.js",
+      "lib/execution-info.js",
```

Comment:
Should be `lib/execution-options.js` now

Reply:
Good catch!

I've created a ticket to generate docs in Jenkins for this type of validation: https://datastax-oss.atlassian.net/browse/NODEJS-478